SCClassLibrary/Common/Control/asGroup.sc

```diff
@@ -4,12 +4,12 @@
 
 + Nil {
 	asGroup {
-		^Group.basicNew(this, 1)
+		^Server.default.defaultGroup
```

Comment:
Generally, because hardcoding is almost never a good idea, and more specifically, because the default server may have a different defaultGroup than 1, and the error will be hard to find then.